README.md

```diff
@@ -323,6 +323,39 @@ service EchoService {
  }
 ```
 
+### 协议嗅探
+
+一个端口支持同时接受 dubbo 和 kitex 原生协议的请求，只需要在初始化 server 的时候加上组件即可，示例如下：
+```
+import (
+	"github.com/cloudwego/kitex/server"
+	dubbo "github.com/kitex-contrib/codec-dubbo/pkg"
+	hello "demo-server/kitex_gen/hello/greetservice"
+	"log"
+	"net"
+)
+
+func main() {
+	// 指定服务端将要监听的地址
+	addr, _ := net.ResolveTCPAddr("tcp", ":21000")
+	svr := hello.NewServer(new(GreetServiceImpl),
+		server.WithServiceAddr(addr),
+		// 配置 DubboCodec
+		server.WithTransHandlerFactory(dubbo.NewSvrTransHandlerFactory(
```

Comment:
I think we should use
```
detection.NewSvrTransHandlerFactory(netpoll.NewSrvTransHandlerFactory, 
dubbo.NewSvrTransHandlerFactory(),
nphttp2.NewSvrTransHandlerFactory(),
)
```
to inject it.
Refer to https://github.com/cloudwego/kitex/blob/a74dfc289a3805d65f7345257d80743c1f2b6523/internal/server/remote_option.go#L35.

Reply:
Done